pkg/utils/stream_fields_parser_test.go

```diff
@@ -0,0 +1,106 @@
+package utils
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestParseStreamFields(t *testing.T) {
+	tests := []struct {
+		name         string
+		streamFields string
+		want         []StreamField
+		wantErr      bool
+	}{
+		{
+			name:         "empty",
+			streamFields: ``,
+			want:         []StreamField(nil),
+			wantErr:      false,
+		},
+		{
+			name:         "incorrect stream field",
+			streamFields: `{"a":1}`,
+			want:         []StreamField(nil),
+			wantErr:      true,
+		},
+		{
+			name:         "empty value field",
+			streamFields: `{b=""}`,
+			want:         []StreamField(nil),
+			wantErr:      true,
+		},
+		{
+			name:         "incorrect value field without quotes",
+			streamFields: `{b=a}`,
+			want:         []StreamField(nil),
+			wantErr:      true,
+		},
+		{
+			name:         "empty label field in quotes",
+			streamFields: `{""="b"}`,
```

Comment:
have you checked if vlogs can have `""` as stream key?

Reply:
removed this tes